components/mamo_business/common/utils.mjs

```diff
@@ -0,0 +1,6 @@
+export const parseObject = (obj) => {
+  if (typeof obj != "object") {
+    return JSON.parse(obj);
```

Comment:
A `try/catch` block with a `ConfigurationError` (if the value has to be an object; if not, the exception could be ignored) would be helpful

Reply:
It must be an object, but if the client fills it in as a custom expression, it comes as a string, so I need to parse it before sending